internal/schema/schema_test.go

```diff
@@ -873,3 +873,137 @@ Second line of the description.
 		})
 	}
 }
+
+func TestInterfaceImplementsInterface(t *testing.T) {
+	for _, test := range []struct {
+		name                  string
+		sdl                   string
+		useStringDescriptions bool
+		validateError         func(err error) error
```

Comment:
Could you, please, replace 
```go
validateError func(err error) error
```

with just:
```go
expectedError string
```
This way at the end of each test we'll be comparing the received error with the expected one.

Reply:
I just noticed that the previous tests are the same. I'll take care of this later and will leave the test for consistency the way you've written it.